geode-connectors/src/acceptanceTest/java/org/apache/geode/connectors/jdbc/JdbcDistributedTest.java

```diff
@@ -259,18 +259,19 @@ public void verifyDateToDate() throws Exception {
       Connection connection = DriverManager.getConnection(connectionUrl);
       Statement statement = connection.createStatement();
       statement.execute(
-          "Create Table " + TABLE_NAME + " (id varchar(10) primary key not null, mydate date)");
+          "Create Table " + TABLE_NAME + " (id varchar(10) primary key not null, "
+              + TestDate.DATE_FIELD_NAME + " date)");
     });
     createRegionUsingGfsh(true, false, true);
     createJdbcConnection();
-    createMapping(REGION_NAME, DATA_SOURCE_NAME);
+    createMapping(REGION_NAME, DATA_SOURCE_NAME, TestDate.class.getName());
     final String key = "emp1";
     final java.sql.Date sqlDate = java.sql.Date.valueOf("1982-09-11");
     final Date jdkDate = new Date(sqlDate.getTime());
     server.invoke(() -> {
       PdxInstance pdxEmployee1 =
```

Comment:
nice to change the variable name to testDate as its not representing employee type

Reply:
done